argh_derive/src/parse_attrs.rs

```diff
@@ -160,8 +160,10 @@ impl FieldAttrs {
             _ => {}
         }
 
-        if let Some(d) = &this.description {
-            check_option_description(errors, d.content.value().trim(), d.content.span());
+        if !type_attrs.lax_descriptions {
+            if let Some(d) = &this.description {
+                check_option_description(errors, d.content.value().trim(), d.content.span());
```

Comment:
I think we should change the behavior of check_option_description rather than skipping it entirely when the option is set.